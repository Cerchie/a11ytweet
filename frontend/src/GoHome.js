import React from 'react'

function GoHome() {
    return (
        <>
            <p style={{ margin: '1rem' }}>
                Success! You've edited your account.
            </p>
            <a href="/" style={{ margin: '1rem' }}>
                Return home.{' '}
            </a>
        </>
    )
}

export default GoHome
