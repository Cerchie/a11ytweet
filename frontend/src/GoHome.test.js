import { render, screen } from '@testing-library/react'
import React from 'react'
import GoHome from './GoHome'

test('renders app', () => {
    render(<GoHome />)
})
