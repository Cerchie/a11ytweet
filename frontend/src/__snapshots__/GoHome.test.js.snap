// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Success! You've edited your account.
    <a
      href="/"
    >
      Return home. 
    </a>
  </div>
</DocumentFragment>
`;
